.github/workflows/Deploy.yaml

```diff
@@ -27,49 +31,99 @@ on:
         type: boolean
         description: Create a release in this repository
         required: false
-        default: true
+        default: false
       defaultBcContainerHelperVersion:
         description: 'Which version of BcContainerHelper to use? (latest, preview, private, a specific version number or a direct download URL like https://github.com/freddydk/navcontainerhelper/archive/master.zip). Leave empty to use latest (or preview for preview branches)'
         required: false
-        default: ''
+        default: 'latest'
+        
+permissions:
```

Comment:
Assign the needed permissions on job-level. I suppose not all jobs need all the permissions.